README.md

```diff
@@ -13,66 +13,113 @@ Assumptions:
 
-| ENV VAR        | VALUE                            |
-|----------------|----------------------------------|
-| USASPENDING_API| https://api.usaspending.gov/api/ |
-| MAPBOX_TOKEN   | ''                               |
-| GA_TRACKING_ID | ''                               |
+| ENV VAR           | DEFAULT VALUE (if not set)       | PURPOSE 
+|-------------------|----------------------------------|----------------------------------------------------|
+| `ENV`             | dev                              | Determine bundling optimizations and feature flags | 
+| `USASPENDING_API` | https://api.usaspending.gov/api/ | API URL to get backend data                        |
+| `MAPBOX_TOKEN`    | ''                               | Product key for use of MapBox features             |
+| `GA_TRACKING_ID`  | ''                               | Google Analytics key for tracking usage in prod    |
 
-### Scripts
+These values can be set in one of 3 methods:
+
+1. In a git-ignored `.env` file, if using `docker` or `docker-compose` to execute scripts 
 
-| Script     | Output                                              |
-|------------|-----------------------------------------------------|
-| `npm start`  | dev server with hot reloading                       |
-| `npm prod`   | generates static assets w/ production optimization  |
-| `npm dev`    | generates static assets w/ development optimization |
-| `npm lint`   | executes linter                                     |
-| `npm test`   | executes unit tests                                 |
-| `npm sass`   | dev server w/ scss source maps                      |
-| `npm travis` | executes travis validation script                   |
-
-### Deployment Build Step with DockerFile
-When deploying we use the dockerfile for our build step. We can simulate that process as follows:
-
-```
-# from usaspending-website root, build the container image
-docker build -t usaspending-website .
-# generate static artifacts in ./public
-docker run -i --rm=true -v $(pwd)/public:/node-workspace/public usaspending-website /bin/sh -c 'npm run dev'
-# mount the static artifacts to an nginx image and run
-docker run -p 8020:80 -v $(pwd)/public:/usr/share/nginx/html:ro nginx
-```
-
-The app should now be running at `http://localhost:8020`.
+    ```bash
+    # .env file
+    USAPENDING_API=http://localhost:8000/api
+    # ... next var
+    ```
+    - Confirm Docker Compose picks up vars by running: `docker-compose config` to see the values applied
+    - Also can be applied with `docker run --env-file .env ...`
+    - _NOTE: Default env var values are set in the `Dockerfile` and will be used in `docker run` if none are provided_
+2. Using `export` to set them in the current executing shell environment.
```

Comment:
This should be #1. As a front end developer, we dont use docker. Our app isn't configured to respect the .env file. This is a docker thing.

Reply:
I can change. It changes the statement about precedent, but we'll be moving that out anyway.